### PR TITLE
Honor explicit dependencies (notion: fixed) OKAPI-946

### PR DESCRIPTION
**Pull request by adamdickmeiss**

### CODE REVIEW

File: okapi-core/src/main/java/org/folio/okapi/bean/TenantModuleDescriptor.java [hunk 1]
```diff
@@ -33,6 +33,10 @@
     pending, deploy, invoke, undeploy, done
   }
 
+  // if true, it specifies that the id is specified by user and can not be changed
```
Comment (julianladisch) [inline okapi-core/src/main/java/org/folio/okapi/bean/TenantModuleDescriptor.java:36]: This comment should be reworded, converted into javadoc and attached to the public methods getFixed() and setFixed(Boolean fixed).

`Returns true if the version is specified by user and the product can not be upgraded or downgraded.`

File: okapi-core/src/main/java/org/folio/okapi/bean/TenantModuleDescriptor.java [hunk 2]
```diff
@@ -33,6 +33,10 @@
...
+  @JsonIgnore
+  private Boolean fixed;
```
Comment (julianladisch) [inline okapi-core/src/main/java/org/folio/okapi/bean/TenantModuleDescriptor.java:38]: I don't see the need why this should have three values: true, false, null. Can a boolean (with true and false only) been used instead?

**Review by julianladisch — APPROVED** [review]
Looks good, thanks!

File: okapi-core/src/main/java/org/folio/okapi/bean/TenantModuleDescriptor.java (hunk 2 above)
Comment (adamdickmeiss) [reply okapi-core/src/main/java/org/folio/okapi/bean/TenantModuleDescriptor.java:38]: True. I'll change that.

**Pull request merged**